rust/tests/common.rs

```diff
@@ -9,3 +9,12 @@ pub fn map_mcap<P: AsRef<Utf8Path>>(p: P) -> Result<Mmap> {
     let fd = fs::File::open(p).with_context(|| format!("Couldn't open {p}"))?;
     unsafe { Mmap::map(&fd) }.with_context(|| format!("Couldn't map {p}"))
 }
+
+#[allow(dead_code)]
+pub fn mcap_test_file() -> Result<Mmap> {
+    if cfg!(feature = "zstd") {
+        return map_mcap("tests/data/compressed.mcap");
```

Comment:
Nit: Rust automatically returns the last expression in a scope (a la Ruby, Haskell, etc.), so this could just be `map_mcap("tests/data/compressed.mcap")`

Reply:
Thanks good catch. I'm still getting comfortable with Rust. (but enjoying it quite a bit)